src/amp.js

```diff
@@ -39,6 +40,9 @@ import {maybeValidate} from './validator-integration';
 try {
   // Should happen first.
   installErrorReporting(window);  // Also calls makeBodyVisible on errors.
+  let perf = performanceFor(window);
+
+  perf.tick('installStyles');
```

Comment:
done. added TICKEVENTS.md at root dir. will merge tomorrow morning if you have no further comments with the latest changes.